### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HarshDeep61034**
My first Contribution

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2876"></a>

### CODE REVIEW

**Comment by HarshDeep61034** [comment]
I have tried to improve the readme documentation wherever i felt this could be improved

File: README.md [hunk 1]
```diff
@@ -4,22 +4,22 @@
...
-You have 4 options to contribute to the repo, please pick your favourite from:
+You have 4 ways to contribute to this repo, please pick your favourite from:
 
 1. GitHub UI (recommended for adding/editing your profile)
 2. Gitpod
 3. Local development
 4. Local development with Docker Compose
 
-Brief documentation is below, but full documentation can be found here https://linkfree.eddiehub.io/docs
+Brief documentation is below, for full documentation checkout: https://linkfree.eddiehub.io/docs
```
Comment (eddiejaoude) [inline README.md:22]: `checkout` is not a good word to use as it has a technical meaning and could cause confusion.

File: README.md [hunk 2]
```diff
@@ -71,7 +71,8 @@ This will allow you to run your favourite IDE but not have to install any depend
 
 ### 🙂 How to add YOUR Profile
 
-Step by step quickstart guide can be found in the full docs here https://linkfree.eddiehub.io/docs/quickstart
+Step by step quickstart guide is given in the full docs here https://linkfree.eddiehub.io/docs/quickstart
+
```
Comment (eddiejaoude) [inline README.md:75]: Extra line is not required here

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
I left some inline comments

File: README.md (hunk 1 above)
Comment (HarshDeep61034) [reply README.md:22]: sure i appreciate your feedback.

**Comment by eddiejaoude** [comment]
Hi @HarshDeep61034 did you mean to close this pull request?

**Comment by eddiejaoude** [comment]
If you have any questions we have an EddieHub Discord you can join http://discord.eddiehub.org

**Pull request closed without merging**